lib/admin/actions/admin.js

```diff
@@ -47,6 +48,10 @@ export const setUserPage = createAction(
   'SET_USER_PAGE',
   (payload: number) => payload
 )
+export const setUserPerPage = createAction(
+  'SET_USER_PER_PAGE',
```

Comment:
Echoing my suggestion from https://github.com/ibi-group/datatools-server/pull/353#discussion_r522168925 to rename `perPage` and `*_PER_PAGE` to `itemsPerPage` and `*_ITEMS_PER_PAGE`, respectively.

Reply:
Going to leave perPage in place to simplify the query param stringify, but happy to change the action name.